### DISCUSSION TITLE
Ribbon buttons arrangement

### DISCUSSION THREAD

**Discussion by Nice3point** [jeremytammik/RevitLookup#159] [General]
Suggest and vote for your favourite RevitLookup button layout

**Comment by Nice3point** [comment]
Drop-down list
![изображение](https://user-images.githubusercontent.com/20504884/234645624-f8c67a34-5400-4016-a678-ed7935817466.png)

**Reply by chuongmep** [reply]
I voted for this, look simple and easy than to click

**Comment by Nice3point** [comment]
Drop-down list with selected button saved as active
![изображение](https://user-images.githubusercontent.com/20504884/234645049-bc19404c-32e9-4fa8-a898-19e3262ca220.png)

**Comment by Nice3point** [comment]
Drop-down list with grouping
![изображение](https://user-images.githubusercontent.com/20504884/234645370-558248df-7d67-4b0c-9886-b99a627c043f.png)

**Comment by Nice3point** [comment]
New tab, buttons arranged from left to right
![изображение](https://user-images.githubusercontent.com/20504884/234644756-f162ed8d-59e2-4461-96f0-687336c05932.png)
